chsdi/models/vector/uvek.py

```diff
@@ -748,6 +748,62 @@ class FernWaermeAngebot(Base, Vector):
 register('ch.bfe.fernwaerme-angebot', FernWaermeAngebot)
 
 
+class KomoProjekte(Base, Vector):
+    __tablename__ = 'komo_projekte'
+    __table_args__ = ({'schema': 'bfe', 'autoload': False})
+    __template__ = 'templates/htmlpopup/komo.mako'
+    __bodId__ = 'ch.bfe.komo-projekte'
+    __queryable_attributes__ = ['project_title_de', 'project_title_fr', 'project_title_it', 'project_title_en']
+    __label__ = 'id'
+    id = Column('bgdi_id', Integer, primary_key=True)
+    project_title_de = Column('project_title_de', Unicode)
+    project_title_fr = Column('project_title_fr', Unicode)
+    project_title_it = Column('project_title_it', Unicode)
+    project_title_en = Column('project_title_en', Unicode)
+    projectstart = Column('projectstart', Integer)
+    projectend = Column('projectend', Integer)
+    amount_sponsored = Column('amount_sponsored', Float)
+    total_cost = Column('total_cost', Float)
+    tid_topic = Column('tid_topic', Integer)
+    web_de = Column('web_de', Unicode)
+    web_fr = Column('web_fr', Unicode)
+    web_it = Column('web_it', Unicode)
+    web_en = Column('web_en', Unicode)
+    mail = Column('mail', Unicode)
+    aim_de = Column('aim_de', Unicode)
+    aim_fr = Column('aim_fr', Unicode)
+    aim_it = Column('aim_it', Unicode)
+    aim_en = Column('aim_en', Unicode)
+    measure_de = Column('measure_de', Unicode)
```

Comment:
attribute measure_* is not referenced in mako, do we really need it in the model?

Reply:
As for erneuerbarheizen, those attributes are needed in the API for external applications. This is an explicit request from the DH